Misc/NEWS.d/next/Library/2018-04-08-22-54-07.bpo-33185.Id-Ba9.rst

```diff
@@ -0,0 +1,5 @@
+Fixed regression when running pydoc with the ``-m`` switch. (The regression
+was introduced in 3.7.0b3 by the resolution of bpo-33053)
```

Comment:
Missed period.

Shouldn't the reference  to other issue be written as ``` :issue:`33053` ```?

Reply:
Bleh, this is the main downside of the switch to separate news snippets - no other surrounding entries to prompt styling fixes.